File: web3rpc/rpc-specs/code-samples/python/klay/block/getCFS.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

w3 = Web3(Web3.HTTPProvider(host))

# klay_getCFS has no dedicated SDK wrapper; call it through the provider.
klay_response = w3.provider.make_request("klay_getCFS", ["latest"])

print(klay_response)

File: web3rpc/rpc-specs/code-samples/javascript/klay/block/getCFS.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(async () => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");

  // klay_getCFS has no dedicated SDK wrapper; call it through the provider.
  const blockNumberOrTag = "latest";
  const result = await provider.send("klay_getCFS", [blockNumberOrTag]);
  console.log(result);
})();
